### PR TITLE
Helm charts release schedule

### PR DESCRIPTION
**Pull request by leandroberetta**
This PR adds the scheduling mechanism on Mondays.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -21,53 +23,111 @@ jobs:
...
         then
-            RELEASE_VERSION=$(./.github/workflows/util/bump.py $RELEASE_TYPE $RELEASE_VERSION)
+          RELEASE_VERSION=$(python bump.py $RELEASE_TYPE $RELEASE_VERSION)
+        elif [[ $RELEASE_TYPE == "minor" ]]
+        then
+          RELEASE_VERSION=$RELEASE_VERSION
         fi
 
-        echo "::set-output name=release_version::$RELEASE_VERSION"      
+        echo "::set-output name=release_version::$RELEASE_VERSION"
```
Comment (jmazzitelli) [inline .github/workflows/release.yml:110]: does this have spaces at the end of line? I see alot of changes recently removing end of line spaces - I assume you missed this one.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,12 +1,14 @@
 name: Release
 
 on:
+  schedule:
+    # Every Friday at 19:00 (UTC)
+    - cron: '00 19 * * FRI'
```
Comment (jmazzitelli) [inline .github/workflows/release.yml:6]: I'm confused with this PR - is this releasing every week via this cron schedule? I'm sure that isn't what its doing, but without comments in here explaining, it is confusing.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (leandroberetta) [reply .github/workflows/release.yml]: I missed that one, yes.

File: .github/workflows/release.yml (hunk 2 above)
Comment (leandroberetta) [reply .github/workflows/release.yml:6]: This PR adds the schedulling mechanism to the Helm charts release pipeline. I'm not sure when you say that it isn't what is doing. 

This PR adds the cron, the release type selection (checking if it needs to skip the release if we are not in the end sprint week) and also adds some checks similar to the server and operator ones.

Comment (leandroberetta) [reply .github/workflows/release.yml:6]: I added the "do not merge" label to wait after the next release that will be manual.

Comment (jmazzitelli) [reply .github/workflows/release.yml:6]: I thought we did this every 3 weeks. This says in the comment it does this every Friday. So I guess it determines later if it is in the third week to actually do the work (the other 2 weeks it just does nothing?)

Comment (leandroberetta) [reply .github/workflows/release.yml:6]: Yes, with a cron sentence you can't fire every 3 weeks, it was commented on the following PR too: https://github.com/kiali/kiali/pull/5037 (comment #2).

So, I don't like the fact that we will see a release pipeline run for nothing, but it's the way GH Actions works today.

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**